runtime/win32.c

```diff
@@ -607,6 +607,11 @@ void caml_win32_overflow_detection(void)
   AddVectoredExceptionHandler(1, caml_stack_overflow_VEH);
```

Comment:
@dra27 Oh, my... `AddVectoredExceptionHandler()` returns an OS handle that should be supplied to `RemoveVectoredContinueHandler()`, so the fix is totally incorrect. Sorry for that. What is better: to add a new commit or to rewrite the existing one? And does it make sense to check `AddVectoredExceptionHandler()`/`RemoveVectoredContinueHandler()` result and call `caml_fatal_error()` if something goes wrong?

Reply:
Oops, so it does! It's up to you whether to push a new commit and have the PR squashed when it's merged, or just rewrite the commit now (personally, I would rewrite the commit).

I wouldn't add error handling to the remove call (other "free"ing operations in `caml_shutdown` don't either).

`caml_win32_overflow_detection` should call `caml_fatal_error` if the handler fails to install though (it's either that or `caml_win32_unregister_overflow_detection` should not pass `NULL` to the API function)